test/Altinn.Platform.Authentication.Tests/RepositoryDataAccess/SystemUserRepositoryDbTests.cs

```diff
@@ -45,14 +55,21 @@ public async Task InsertSystemUser()
     [Fact]
     public async Task GetAllActiveSystemUsersForParty()
```

Comment:
i think all the tests can have a summary that defines the test case

Reply:
Ok